Cargo.toml

```diff
@@ -38,12 +38,26 @@ wasm-bindgen = { version = "0.2.82", features = [
 ] } # rustwasm/wasm-bindgen#2963
 web-sys = { version = "0.3", features = [
     "CanvasRenderingContext2d",
+    "CssStyleDeclaration",
+    "Document",
+    "Element",
+    "Event",
+    "EventTarget",
     "HtmlCanvasElement",
+    "HtmlCollection",
+    "HtmlElement",
+    "MouseEvent",
     "Navigator",
+    "Node",
+    "NodeList",
```

Comment:
Could you check if this feature `NodeList` is truly required and clarify where it is being used?

Reply:
Thanks for pointing that out! I pushed a small clean-up that drops the unused web-sys feature flags so we only build the pieces we actually use.

Reply:
Understood. I have removed only NodeList and left the rest intact.